packages/preview/kip/0.1.0/README.md

```diff
@@ -0,0 +1,186 @@
+- ✅ No external dependencies - uses WebAssembly plugin
+- ✅ Fast and lightweight (~125KB WASM module)
+- ✅ Works in Typst CLI (tested with Typst 0.12+)
+- ✅ Simple, intuitive API
+
+## Installation
+
+### Option 1: Local Installation
+
+1. Download or clone this package
+2. Place it in your Typst local packages directory:
+   - **Windows**: `%APPDATA%\typst\packages\local\kip\0.1.0\`
+   - **macOS**: `~/Library/Application Support/typst/packages/local/kip/0.1.0/`
+   - **Linux**: `~/.local/share/typst/packages/local/kip/0.1.0/`
+
+### Option 2: Direct Import
+
+Place this directory next to your Typst document and import it directly:
+
+```typst
+#import "kip/lib.typ": kip
+```
+
+## Usage
+
+### Basic Example
+
+````typst
+#import "@local/kip:0.1.0": kip
+
+#kip(```
+box "Start"
+arrow
+circle "End" fit
+```)
+````
+
+### With String Input
+
+```typst
+#kip("box \"Hello\" \n arrow \n circle \"World\" fit")
+```
+
+### With Sizing
+
+````typst
+#kip(
+  ```
+  arrow right 200%
+  box "Process" fit
+  arrow right 200%
+  ```,
+  width: 400pt
+)
+````
+
+## Examples
+
+### Simple Flow
+
+````typst
+#kip(```
+box "Start"
+arrow
+box "Process"
+arrow
+circle "End" fit
+```)
+````
+
+### State Machine
+
+````typst
+#kip(```
+circle "Idle" fit
+arrow right 150% "start" above
+circle "Active" fit
+arrow right 150% "finish" above
+circle "Done" fit
+```)
+````
+
+### Arrow Diagram
+
+````typst
+#kip(```
+arrow right 200% "Input"
+box rad 10px "Processor" fit
+arrow right 200% "Output"
+```)
+````
+
+### Architecture Diagram
+
+````typst
+#kip(```
+box "Frontend" width 3cm fit
+arrow down 50%
+box "API Layer" width 3cm fit
+arrow down 50%
+box "Database" width 3cm fill lightblue fit
+```)
+````
+
+## API Reference
+
+### `kip(code, width: auto, height: auto, fit: "contain")`
+
+Renders a Pikchr diagram from the provided markup code.
+
+**Parameters:**
+- `code` (string or content): The Pikchr markup code (can be a string or raw block)
+- `width` (length, auto): Optional width constraint
+- `height` (length, auto): Optional height constraint
+- `fit` (string): How to fit the image ("contain", "cover", "stretch")
+
+**Returns:** An image element containing the rendered diagram
+
+### Aliases
+
+For backwards compatibility and convenience:
+- `pikchr()` - alias for `kip()`
+- `render()` - alias for `kip()`
+
+All three functions work identically:
+````typst
+#import "@local/kip:0.1.0": kip, pikchr, render
+
+#kip("box \"A\"")      // Primary function
+#pikchr("box \"B\"")   // Alias
+#render("box \"C\"")   // Alias
+````
+
+## Pikchr Language Reference
+
+For complete Pikchr syntax and examples, visit:
+- [Pikchr Homepage](https://pikchr.org)
+- [Pikchr Documentation](https://pikchr.org/home/doc/trunk/homepage.md)
+- [Pikchr Examples](https://pikchr.org/home/doc/trunk/doc/examples.md)
+
+## Building from Source
+
+If you need to rebuild the WASM module (e.g., for updates or modifications), see the [build-scripts/BUILD.md](build-scripts/BUILD.md) file.
```

Comment:
I don't think this is relevant to users of the package on Universe. You should either remove that section or be more explicit that it requires to have cloned the repository first.